### ISSUE TITLE
Page counters in the document

### ISSUE THREAD

**Issue by SimonSapin** [Kozea/WeasyPrint#91]
As discussed on the mailing list, `content: counter(page)` and `content: counter(pages)` should work everywhere, not just in page-margin boxes. This is not implemented yet.

**Comment by mashony** [comment]
Hi @SimonSapin. Are threre any news?

**Comment by SimonSapin** [comment]
See https://github.com/Kozea/WeasyPrint/issues/91#issuecomment-30117473.

**Comment by nvernooy** [comment]
Any update on this?

Using 
```
@page {
    counter-increment: page;
}
#pagesCounter::after {
     display: inline;
     content: counter(pages);
}

This paper consists of <span id="pagesCounter"></span> pages.
```

Results in
`This paper consists of 0 pages`

Using `counter(pages)` in the page margins works perfectly, but I need to use it inside the page area.

**Comment by liZe** [comment]
Fixed by #652.

**Comment by viperthapa** [comment]
I have been stucked it for several hours ..Thank you so much..It solved.
